tests/Feature/Controller/Student/GetStudentImageControllerTest.php

```diff
@@ -9,89 +9,87 @@
 };
 use App\Service\Student\GetStudentImageService;
 use Illuminate\Foundation\Testing\DatabaseTransactions;
-use Mockery\MockInterface;
+use Illuminate\Support\Facades\Storage;
 use Tests\TestCase;
 
 class GetStudentImageControllerTest extends TestCase
 {
     use DatabaseTransactions;
+    private const PHOTOS_PATH = 'public/photos/';
+    protected User $user;
+    protected Student $student;
 
     protected function setUp(): void
     {
         parent::setUp();
-    }
 
-    private function createUser()
-    {
-        return User::factory()->create([
+        $this->user = User::factory()->create([
             'dni'=>'27827083G', 'password'=>'Password%123'
         ]);
+
+        $this->student =Student::factory()->for($this->user)->create();
+
+    }
+
+    public function testCanInstantiateAnUser(): void
```

Comment:
Remove the fixtures tests (only check the instance of what you are testing, if this has sense)